### ISSUE TITLE
Satin columns not saving to pes file 

### ISSUE THREAD

**Issue by annnabanannna** [inkstitch/inkstitch#2048]
Hi, 
I've been making quite small satin columns using the rung and rail method (requiring a satin stitch of around 1-2mm width). The satin stitch looks great when I run it through perams and the embroidery simulator but when I go to stitch the image out on my machine only some of the columns have converted to satin and other objects have come out as the rails and rungs stitched in running stitch. Any ideas why this might be happening? I've checked, re-checked, saved and re-saved the file a number of times with no change. Thanks

**Comment by kaalleen** [comment]
Usually this would happen, if the satin is too wide. But 1-2 mm width is definitely not too wide. It's difficult to say what is going wrong without having a file to look at. Would you mind to send your file to us? You can also isolate the errorneous satin and delete the rest of the design.

**Comment by annnabanannna** [comment]
Thanks for the reply. I realised this was only happing to satin columns that I had not individually run through params. I was selecting a number of objects at once and applying params. It all looked fine in the simulation, but when I went back and opened params on the single objects, they had reverted back to running stitch. Stitching the design out worked fine once I had gone back and changed each of the objects to satin stitch individually.
